proxy/coco/src/peer.rs

```diff
@@ -774,17 +804,20 @@ mod test {
     #[tokio::test]
     async fn can_clone_user() -> Result<(), Error> {
         let alice_key = SecretKey::new();
+        let alice_signer = signer::BoxedSigner::from(signer::SomeSigner {
```

Comment:
A quality of life change might be to have `impl From<SecretKey> for BoxedSigner`

Reply:
Rusts foreign rules would make that impossible to do in upstream. Do you think this should land in link?

Reply:
Ya, that's what I meant :) If we notice annoyances in proxy then we should make changes in link if they make sense.